### ISSUE TITLE
Why do cards have an unconditional box-shadow?

### ISSUE THREAD

**Issue by atu-record** [twbs/bootstrap#19143]
I think it's not very wise to force a mandatory border for cards

at least I would suggest to make it depend on the ```$enable-shadows``` flag

line 12
```
  // Doesn't use mixin so that cards always have a "border"
  box-shadow: inset 0 0 0 $card-border-width $card-border-color;
```

**Comment by cvrebert** [comment]
For reasons currently known only to @mdo, cards currently lack a conventional border (#19097) and have only a box-shadow.

**Comment by mdo** [comment]
It's temporary as I fuck with the basic styles there. Borders are coming back, just messing around and never got back to it.

**Comment by mdo** [comment]
Being addressed in #19861.
